package.json

```diff
@@ -53,9 +53,9 @@
   ],
   "dependencies": {
```

Comment:
is this a major version bump?  I don't see it in in commits with `!` or in the version here?

Reply:
yeah, there's a `feat!` commit in there with BREAKING CHANGES: etc.

Reply:
https://github.com/forcedotcom/sfdx-core/commit/66083f7154a4b9da8f351375150e2c2c5b494bdc